README.md

```diff
@@ -141,7 +141,7 @@ Clone this repository to your desired folder:
 
```

Comment:
**[OPTIONAL REQUIREMENTS]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍